tests/Dfe.EarlyYearsQualification.UnitTests/Mocks/MockContentfulServiceTests.cs

```diff
@@ -123,4 +123,15 @@ public async Task GetStartPage_ReturnsExpectedDetails()
         result!.RightHandSideContentHtml.Should().NotBeNullOrEmpty();
         result!.RightHandSideContentHeader.Should().NotBeNullOrEmpty();
     }
+
+    [TestMethod]
+    public async Task GetPhaseBannerContent_ReturnsExpectedDetails()
+    {
+        var result = await _contentfulService.GetPhaseBannerContent();
+        result.Should().NotBeNull();
+        result.Should().BeAssignableTo<PhaseBanner>();
+        result!.ContentHtml.Should().NotBeNullOrEmpty();
+        result!.PhaseName.Should().NotBeNullOrEmpty();

```

Comment:
suggestion: no need to assert it's not null (`!` is redundant) except the first time

[the same elsewhere in this class]

Reply:
Resolved all these in the class now